test/jdk/java/util/Formatter/BasicInt.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
It looks line the non-float/double test classes are unchanged, they could be dropped from the PR.

Reply:
The only changes I made myself in the test files are in Basic-X.java.template (including the copyright year). The other files were generated by a script, which happens to also change the copyright year for otherwise unmodified files.

If you agree, I'll happily remove the trivial copyright year changes and only retain those files containing more substantial modifications.

Reply:
Fine with me, I figured it was the script that changed the copyright and otherwise generated the same as before.  tnx

Reply:
Otherwise untouched files are now reverted back with previous copyright year